recipes/yarn.rb

```diff
@@ -171,6 +180,14 @@
     mode "1755"
     dest "/user/#{node['hadoop_spark']['user']}/#{hsfs_utils_java}"
   end
+
+  hops_hdfs_directory "#{Chef::Config['file_cache_path']}/#{hopsworks_jobs_py}" do
```

Comment:
If you see at the end of this file, there is a section `if is_head_node`. You need to add this python script in that list. It's needed for the cloud so that we can put it on S3 when deploying a new instance.